docs/search/3_schema_tips_and_tricks.md

```diff
@@ -0,0 +1,43 @@
+# Practical tips for working with our Vespa schemas
+
+The Vespa schema is written in a proprietary language, and a lot of its more useful features don't jump straight out of the docs. This is a page for tips and tricks for working with Vespa schemas.
+
+## Field analysis
```

Comment:
might be worth highlighting here that we've been caught out by the field analysis not being applied to search terms before! easy to assume that an augmentation of the indexed terms will be automatically applied to incoming search terms, but it doesn't seem to be the case in vespa!

Reply:
are you talking about exact match? if so, i think that case was the opposite: stemming _was_ applied to indexed text but we didn't realise

Reply:
ah right, i had misremembered the details! 
all the more reason to document it?

Reply:
[how does this look to you](https://github.com/climatepolicyradar/cpr-sdk/pull/181/commits/53d154f9f0af3a7f657fe1e13cf5f00aa0154b62)?